lumber-install.js

```diff
@@ -11,6 +11,10 @@ const argv = require('minimist')(process.argv.slice(2));
 
 program
   .description('Install a Lumber plugin')
+  .option('-e, --email <email>', 'Your Forest Admin account email')
+  .option('-P, --password <password>', 'Your Forest Admin account password')
+  .option('-t, --token <token>', 'Your Forest Admin account token')
```

Comment:
Is that an authentication token for your forst admin password?

Are -P and -t mutually exclusive? If so, it might be useful to state it.

Reply:
@Sidnioulz 

> Are -P and -t mutually exclusive? If so, it might be useful to state it.

Yes, good catch! Not sure how to give this info though. Do you have any suggestion?

Reply:
Something like that:

* -f, --foo <foo>, 'Does foo. Ignored if bar is set.'
* -b --bar <bar>, 'Does bar. When both foo and bar are set, foo is ignored.'

Reply:
Thank you! Fixed! (I adapted your text to reflect the actual behavior (which is a bit strange BTW))